domain/beneficiary_pre_subscription/beneficiary_pre_subscription_exceptions.py

```diff
@@ -1,2 +1,10 @@
 class CantRegisterBeneficiary(Exception):
     pass
+
+
+class BeneficiaryIsADupplicate(CantRegisterBeneficiary):
```

Comment:
Il y a un p en trop :)
```suggestion
class BeneficiaryIsADuplicate(CantRegisterBeneficiary):
```